examples/LossMinimization/loss_minimization.jl

```diff
@@ -59,7 +62,15 @@ initial_ensemble = EKP.construct_initial_ensemble(rng, prior, N_ensemble)
 # We then initialize the Ensemble Kalman Process algorithm, with the initial ensemble, the
 # target, the stabilization and the process type (for EKI this is `Inversion`, initialized 
 # with `Inversion()`). 
-ensemble_kalman_process = EKP.EnsembleKalmanProcess(initial_ensemble, G_target, Γ_stabilization, Inversion())
+ensemble_kalman_process = EKP.EnsembleKalmanProcess(
+    initial_ensemble,
+    G_target,
+    Γ_stabilization,
+    Inversion(),
+    scheduler = DefaultScheduler(1),
+    accelerator = DefaultAccelerator(),
+    localization_method = EnsembleKalmanProcesses.Localizers.NoLocalization(),
+)
```

Comment:
Ollie - these changes are just to prevent new defaults from kicking in correct?

Reply:
Yes, good spot, I added this line into the docs.